frontend/javascripts/libs/user_local_storage.js

```diff
@@ -0,0 +1,32 @@
+// @flow
+
+import Store from "oxalis/store";
+
+function prefixKey(key) {
+  const { activeUser } = Store.getState();
+  console.log("activeUser", activeUser);
+  const prefix = !activeUser ? "Anonymous" : activeUser.email;
+  return `${prefix}-${key}`;
+}
+
+const UserLocalStorage = {
+  getItem(key: string): ?string {
+    console.log("getting", key);
+    return localStorage.getItem(prefixKey(key));
+  },
+
+  setItem(key: string, value: string): void {
+    console.log("setting", key);
+    return localStorage.setItem(prefixKey(key), value);
+  },
+
+  clear(): void {
+    localStorage.clear();
+  },
+
+  removeItem(key: string): void {
+    return localStorage.removeItem(key);
```

Comment:
Shouldn't this also use `prefixKey(key)`?

Reply:
Whoops, well spotted! :+1: